doc/cephfs/administration.rst

```diff
@@ -234,6 +234,45 @@ For example, to only allow Nautilus clients, use:
 
 Clients running an older version will be automatically evicted.
 
+Enforcing minimum version of CephFS client is achieved by setting required
+client features. Commands to manipulate required client features of a file
+system:
```

Comment:
Since we can also use "nautilus" (and I presume "octopus") as "features", we should make that clearer here.

How about "You can also specify specific required features, rather than a whole Ceph release, which is especially useful when working with kernel clients that may not implement all of the server release's features as a block. Commands to manipulate..." or something like that?